docs/apm/storage-explorer.asciidoc

```diff
@@ -0,0 +1,76 @@
+
+beta::[]
+
+Analyze your APM data and manage costs with **storage explorer**.
+For example, analyze the storage footprint of each of your services to see which are producing
+large amounts of data--then change the sample rate of a service to lower the amount of data ingested.
+Or, expand the time filter to visualize data trends over time so that you can better forecast
+and prepare for future storage needs.
+
+[role="screenshot"]
+image::apm/images/storage-explorer-overview.png[APM Storage explorer]
+
+[float]
+==== Index lifecycle phases
+
+A default {apm-guide-ref}/ilm-how-to.html[index lifecycle policy] is applied to each APM data stream,
+but can be customized depending on your business needs.
+Use the **Index lifecycle phase** dropdown to visualize and analyze your storage by phase.
+
+Customizing the default APM index lifecycle policies can save money by specifying things like:
+
+* The point at which an index can be moved to less performant hardware.
+* The point at which availability is not as critical and the number of replicas can be reduced.
+* When the index can be safely deleted.
+
+See {apm-guide-ref}/ilm-how-to.html[Index lifecycle management] to learn more about customizing
+the default APM index lifecycle policies.
+
+[float]
+==== Service statistics
+
+The service statistics chart provides detailed information on each service:
+
+* A list of **service environments**.
+* The **sampling rate**. This value is calculated by dividing the number of sampled transactions by total throughput.
+It might differ from the configured sampling rate for two reasons: with head-based sampling,
+the initial service makes the sampling decision, and with tail-based sampling,
+granular policies allow you to set multiple sample rates.
+* The estimated **size on disk**. This storage size includes both primary and replica shards and is
+calculated by prorating the total size of your indices by the service's document count divided by
+the total number of documents.
+* Number of **transactions**, **spans**, **errors**, and **metrics** — doc count and size on disk.
+
+[role="screenshot"]
+image::apm/images/storage-explorer-expanded.png[APM Storage explorer service breakdown]
+
+As you explore your service statistics, you might want to take action to reduce the number of
+documents and therefore storage size of a particular service.
+
+[float]
+===== Reduce the number of transactions
+To reduce the number of transactions a service generates, configure a more aggressive
+{apm-guide-ref}/sampling.html[transaction sampling policy]. Transaction sampling lowers
+the amount of data ingested without negatively impacting the usefulness of your data.
+
+[float]
+===== Reduce the number of spans
+To reduce the number of spans a service generates, enable
+{apm-guide-ref}/span-compression.html[span compression]. Span compression saves on data
+and transfer costs by compressing multiple, similar spans into a single span.
+
+[float]
+===== Reduce the number of metrics
```

Comment:
These are both system metrics and calculated metrics which are used to power visualisations in APM. There are ways to reduce the number of documents, for example, by adjusting the [metrics interval](https://www.elastic.co/guide/en/apm/agent/java/current/config-reporter.html#config-metrics-interval) for system metrics in the APM agents or changing the [transaction metrics interval](https://www.elastic.co/guide/en/apm/guide/current/transaction-metrics.html#transactions-interval) in the APM server, but I am not sure if this is the right thing to suggest here. cc @elastic/apm-server

Reply:
I would focus on the system, runtime, and application metrics; and avoid making any recommendations about tuning the interval for transaction metrics for now. We _could_ suggest increasing the transaction metrics interval, but then we need to highlight the impact on time to glass. One minute is already a bit of a wait.

For the system, runtime, and application metrics (i.e. anything captured by agents), we can also suggest the [`disable_metrics`](https://www.elastic.co/guide/en/apm/agent/java/current/config-reporter.html#config-disable-metrics) configuration.